docs/contributing.md

```diff
@@ -39,7 +39,14 @@ by the operator and mounted in the Agent's Pod.
 Each Pod holds a member of a Replica Set, and each Pod has different components,
 each one of them in charge of some part of the lifecycle of the MongoDB database.
 
-# Developing locally
+# Getting Started
```

Comment:
[q] do we have in this doc also a note about signing commits? We had a PR merged today in which the contributor had to rebase after setting up signing, we should put is as prerequisites to make the process smother

Reply:
Good point, will add something about it!